lib/app.js

```diff
@@ -485,6 +483,24 @@ var makeApp = function(configBase, callback) {
 
         app.use(xssCheck.xssCheck);
 
+        if (config.noweb) {
+            // Not serve content of 'javascript' or 'template' folder in noweb,
+            // this before serve publics for prevent response
+            // and after other middlewares to let them work well
```

Comment:
Can you change this comment to:

> Don't serve content of `javascript` or `template` directories in noweb.
>
> This is before `express.static()` so we can override its behavior.

I think that's a bit clearer? What do you think/

Reply:
Yes, you have better english than me :D